docs/en/sql-reference/functions/other-functions.md

```diff
@@ -2499,3 +2499,39 @@ Result:
 │                      286 │
 └──────────────────────────┘
 ```
+
+## enumerateStreams {#enumerateStreams}
+
+return the enumerated stream paths of data type.
```

Comment:
This is very not obvious description for user.
However, this function may be somehow useful to show all the files which are created to serialize column with this type. Like, `enumerateStreams('Array(Array(Int8))')` -> `['.bin','.size0.bin','.size1.bin']`

The name is not clear as well. Maybe something like `getDataTypeSerializationPaths`.

Or maybe even
```
getDataTypeSerializationPaths(my_column) -> 
[
(['ArraySizes'], 'my_column.size0.bin', 'my_column.size0.mrk2'),
(['ArrayElements', 'ArraySizes'], 'my_column.size1.bin', 'my_column.size1.mrk2')
(['ArrayElements', 'ArrayElements', 'Regular'], 'my_column.bin', 'my_column.mrk2')
]
```

Reply:
@KochetovNicolai I create this function to make it convenient for clickhouse developers instead of common users viewing substreams because it is not obvious. 

As for `getDataTypeSerializationPaths`, I think it is obvious enough because we can get them in filesystem. Besides, its results depends on the type of MergeTree data part.

Put column as arguments is a good idea, I will try.

Reply:
done

Reply:
ok, but I don't like the name `enumerateStreams` anyway. It is a function for developers, but it's almost impossible to understand what it does just by it's name. `getTypeSerializationStreams`, or `getDataTypeSerialization` or something like that

Reply:
ok, I will rename it.